public/partials/farmer_profile_large.html

```diff
@@ -98,119 +98,130 @@ <H1 class="farmerSummary">{{farmer.fa_first_name}} {{farmer.fa_middle_name}} {{f
-                            </div>
-                            <div layout="column" flex>
-                                <div layout-align="start center" layout="column">
-                                    {{farm.fr_size}}
-                                    <div class="label">Farm Size (Acre)</div>
-                                </div>
-                            </div>
-                            <div layout="column" flex>
-                                <div layout-align="start center" layout="column">
-                                    {{farm.ad_address1}}
-                                    <div class="label">Address 1</div>
-                                </div>
-                            </div>
+                    <div flex layout="row">
+                        <md-icon class="no-margin" md-svg-icon="/images/icons/icons_variety.svg"></md-icon>
+                        <div layout-align="center" style="text-align: center;" layout="column" ng-class="{'md-offset': 1}">
+                            {{isValid(commodity.cr_crop.cr_crop_variety) ? commodity.cr_crop.cr_crop_variety:  'N/A'}}
+                            <div class="label">Variety</div>
                         </div>
-                        <div layout-align="space-around start" layout-padding layout="row">
-                            <span flex="5"></span>
-                            <div layout="column" flex>
-                                <div layout-align="start center" layout="column">
-                                    {{farm.ad_city}}
-                                    <div class="label">City</div>
-                                </div>
-                            </div>
-                            <div layout="column" flex>
-                                <div layout-align="start center" layout="column">
-                                    {{farm.di_district.di_parish_name}}
-                                    <div class="label">Parish</div>
-                                </div>
-                            </div>
-                            <div layout="column" flex>
-                                <div layout-align="start center" layout="column">
-                                    {{farm.di_district.di_extension_name}} - {{farm.di_district.di_district_name}}
-                                    <div class="label">Extension - District</div>
-                                </div>
-                            </div>
+                    </div>
+
+                    <div flex layout="row">
+
+                    </div>
+
+                </div>
+
+
+                <div layout-align="space-around start" class="farmer_info" layout="row">
+
+
+
+                    <div flex layout="row">
+                        <md-icon class="no-margin" md-svg-icon="/images/icons/icons_amount.svg"></md-icon>
+                        <div layout-align="center" style="text-align: center;" layout="column" ng-class="{'md-offset': 1}">
```

Comment:
@nick22891 please remember the less you do in manipulating the style of a prebuilt object the easier it will be to maintain.